servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/QuarkusBigTableConfig.java

```diff
@@ -42,6 +44,19 @@ public interface QuarkusBigTableConfig {
 
   Map<String, String> jwtAudienceMapping();
 
+  @WithDefault("PT0.2S")
+  Duration maxRetryDelay();
```

Comment:
Not sure we gain much by making `maxRetryDelay` configurable while `initialRetryDelay` is not. The default for `initialRetryDelay` is `0`, so `maxRetryDelay` will never come into playm if I read the javadoc on `RetrySettings` correctly... WDYT?

Reply:
Did you mean to make `initialRpcTimeout()` configurable?

Reply:
Added that, but left maxRetryDelay

Reply:
Hmmm, I still think we need `initialRetryDelay` otherwise retry delays will always be `0`.

Reply:
Line 49??

Reply:
that's `initialRpcTimeout`, not delay.

I wonder if we could have the whole `RetrySettings` configurable via Quarkus... It's got a lot of fine-tuning options... WDYT?

Reply:
Meh - then you'd want to configure it per op-type.
Let's keep it like this for now.